### PR TITLE
Add User limits to TS SDK

### PR DESCRIPTION
**Pull request by taiyosogawa**
### Changes proposed: 
- Adds ability to query user limits to the TS SDK

In addition to the included unit test, I tested this out by consuming it in a local node project.

### CODE REVIEW

File: ts/src/management/tunnelManagementHttpClient.ts [hunk 1]
```diff
@@ -313,6 +315,16 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
         return result;
     }
 
+    public listUserLimits(): Promise<NamedRateStatus[]> {
```
Comment (IlyaBiryukov) [inline ts/src/management/tunnelManagementHttpClient.ts:318]: nit: `async` is easier to read then `.then` chaining.

File: ts/src/management/tunnelManagementClient.ts [hunk 2]
```diff
@@ -155,6 +156,11 @@ export interface TunnelManagementClient {
         options?: TunnelRequestOptions,
     ): Promise<boolean>;
 
+    /**
+     * Lists details of tunneling service clusters in all supported Azure regions.
```
Comment (IlyaBiryukov) [inline ts/src/management/tunnelManagementClient.ts:160]: Please fix the comment (copy-pasted?)

**Review by IlyaBiryukov — APPROVED** [review]

File: ts/src/management/tunnelManagementHttpClient.ts (hunk 1 above)
Comment (taiyosogawa) [reply ts/src/management/tunnelManagementHttpClient.ts]: Done

File: ts/src/management/tunnelManagementClient.ts (hunk 2 above)
Comment (taiyosogawa) [reply ts/src/management/tunnelManagementClient.ts]: Thanks! Done

**Review by IlyaBiryukov — APPROVED** [review]

**Pull request merged**